addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js

```diff
@@ -112,6 +116,10 @@ class StudyService extends Service {
         throw this.boom.forbidden(`Not authorized to add study related to project "${projectId}"`);
       }
       await projectService.mustFind(requestContext, { id: rawData.projectId });
+      // Verify user is not trying to create resources for non-Open data studies
+      if (rawData.resources && rawData.resources.length > 0) {
+        throw this.boom.badRequest('Resources can only be assigned to Open Data study category');
+      }
```

Comment:
There should be test which verifies that any non-open-data request with resources is rejected.

Reply:
Done